react/src/components/organisms/TabContainer/tab.js

```diff
@@ -31,15 +35,53 @@ class Tab extends React.Component {
             this.defaultSet = true;
           }
           return(
-            <button
-              className={tabClasses}
-              onClick={(e) => {
-                this.scrollIntoView(e.target);
-                setActiveTab(this.tabIdent, this.props.children)
-              }}
-            >
-            {this.props.title}
-            </button>
+            <li role="presentation" className={tabClasses}>
+              <button
+                role="tab"
+                id={this.tabIdent}
+                onClick={(e) => {
+                  this.scrollIntoView(e.target);
+                  setActiveTab(this.tabIdent, this.props.children)
+                }}
+                aria-selected={active}
+                tabIndex={!active ? -1 : 0}
+                aria-describedby={this.props.a11yID}
+
+
+    // THIS EVENT IS ONLY AVAILABLE TO THE ONFOCUS TAB.
+                onKeyDown = {(e) => {
+                  if (e.keyCode === 37 || e.keyCode === 39) {
+                    e.preventDefault();
+                    e.stopPropagation();
+
+                    if (this.tabIdent !== document.activeElement.id) {
+                      console.log('Set the active tab!');
+                      setActiveTab(document.activeElement.id, this.props.children);
+                    }
+                  }
+
+                  // TEST OUTPUT
+                  if (e.keyCode === 37) {// LEFT
+                    console.log('LEFT');
+                  }
+                  if (e.keyCode === 39) {// RIGHT
+                    console.log('RIGHT');
+                  }
+                  if (e.keyCode === 40) {// DOWN
+                    console.log('DOWN');
+                  }
+                  console.log('button: ' + this.tabIdent);
+                  console.log("focus at: " + document.activeElement.id);
+                }}
```

Comment:
```suggestion
                }
```

Reply:
I tried to remove the curly braces, but got this error:
<img width="871" alt="storybook" src="https://user-images.githubusercontent.com/9633303/48210259-f7d4cd80-e344-11e8-8015-231599524fe2.png">